src/tab-controller/tests/functional/TabController.ts

```diff
@@ -27,9 +28,9 @@ registerSuite('TabController', {
 	},
 	'tabs should be changable'() {
 		return getPage(this.remote)
-			.findByCssSelector(`.${css.root}`)
-			.findByCssSelector(`.${css.tabButton}:last-child`)
-			.getProperty('className')
+			.find('css selector', `.${css.root}`)
```

Comment:
What's the reasoning behind switching to `find`?

Reply:
I'm replacing `find` with `findByCssSelector` as you said before that the `find` function is better as it will keep polling until the it reaches timeout.